deployment/ccip/changeset/globals/config.go

```diff
@@ -25,8 +25,6 @@ const (
 	InflightCacheExpiry         = 1 * time.Minute
 	RootSnoozeTime              = 5 * time.Minute
 	BatchingStrategyID          = 0
-	GasPriceDeviationPPB        = 1000
-	DAGasPriceDeviationPPB      = 0
```

Comment:
Removed these from here because these are only currently used in tests, would rather not cause confusion and have them accidentally used in real envs.